db/seeds.rb

```diff
@@ -96,4 +124,20 @@
   end
   puts "\n"
 end
-puts 'Seeding user items complete!'
+puts "Seeding user items complete!\n\n"
+
+# Seed 'StoreItems' table
+puts 'Creating store items with prices...'
+Item.all.each do |item|
+  Store.all.each do |store|
+    dollars = (0.0..9.0).step(1).to_a.sample
+    cents = ((10.0..90.0).step(10).to_a.sample + 9.0)/100
+    StoreItem.create!(
+      store_id: store.id,
+      item_id: item.id,
+      price: dollars + cents
```

Comment:
to ensure an easy-to-read value :) 
```suggestion
      price: (dollars + cents).round(2)
```